spec/views/posts/index_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index Page', type: :feature do
+  before do
+    @user = User.create(name: 'John', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Artist from Kenya.')
+    @post = Post.create(author: @user, title: 'First post', text: 'This is my first post')
+    5.times do |i|
+      Comment.create(text: "This is comment ##{i}", author_id_id: @user.id, post_id_id: @post.id)
+    end
+    visit user_posts_path(@user.id)
+  end
+
+  it 'displays post text' do
+    expect(page).to have_content('This is my first post')
+  end
+
+  it 'displays post title' do
+    expect(page).to have_content('First post')
+  end
```

Comment:
Good job writing tests for the post index page, however, you have missed the following tests:

- Tests that assert you can see the first comments on a post.
- Tests that asserts you can see a section for pagination if there are more posts than fit on the view.
- Tests that redirects to the post's show page when you click on a post.

please consider writing tests for the above in order to satisfy the requirements for this milestone.